StudentManagement/StudentManagement.Console/Student.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StudentManagement.Console
+{
+    public class Student
+    {
+        public string Name { get; set; }
```

Comment:
you require to set those properties - Check if you can use: https://stackoverflow.com/a/72443528/4818730

Reply:
I changed set propoerty to init. Now it should be fine. Thanks